### DISCUSSION TITLE
mutt_oauth2.py: personal outlook account ?

### DISCUSSION THREAD

**Discussion by motorto** [neomutt/neomutt#4531] [Development]
Anyone able to connect a personal outlook email using `mutt_oauth2.py` ? 

I was successful connecting my work office365 account but personal email (`outlook.pt`) in my case doesn't work.

```
rm  $XDG_CONFIG_HOME/neomutt/accounts/outlook.token
mutt_oauth2.py --authorize   \
  --encryption-pipe "gpg --encrypt --recipient XXXXXXX"  \ 
  --decryption-pipe="gpg --decrypt" \ 
  --provider microsoft\ 
   --email "personal@outlook.pt" \
    --client-id "20460e5d-ce91-49af-a3a5-70b6be7486d1" \
  --client-secret " " \
   --authflow "authcode" \ 
   $XDG_CONFIG_HOME/neomutt/accounts/outlook.token
```

Client-id stolen from Thunderbird. Purposely adding a space on `client-secret`. Since thunderbird doesn't seem to have one.

**Comment by milanglacier** [comment]
I cannot get it work with outlook email (aka `@outlook.com`)

Whennever I tried to login with the url, it prompted me with the following error:

```
You can't sign in here with a personal account. Use your work or school account instead.
```

The client-id are stolen from the thunderbird:

```
08162f7c-0fd2-4200-a84a-f25a4db0b584
```

**Comment by milanglacier** [comment]
> I cannot get it work with outlook email (aka `@outlook.com`)
> 
> Whennever I tried to login with the url, it prompted me with the following error:
> 
> ```
> You can't sign in here with a personal account. Use your work or school account instead.
> ```
> 
> The client-id are stolen from the thunderbird:
> 
> ```
> 08162f7c-0fd2-4200-a84a-f25a4db0b584
> ```

I successfully configured the script with personal outlook account by using the client id from the app registered by my own.
